pkg/engine/jmespath/functions.go

```diff
@@ -230,13 +230,14 @@ func GetFunctions() []FunctionEntry {
 		FunctionEntry: gojmespath.FunctionEntry{
 			Name: add,
 			Arguments: []argSpec{
-				{Types: []jpType{jpAny}},
-				{Types: []jpType{jpAny}},
+				// {Types: []jpType{jpAny}},
```

Comment:
You can't just change the signature of the function here.

Reply:
so what should be the possible solution? @eddycharly

Reply:
`sum()` would be better, yes.

Reply:
So, you are trying to say that I should leave add function as it is and create a sum function with these new functionality. Am I getting this right @eddycharly  @chipzoller ?

Reply:
Yes, you should just modify this and put everything under a new `sum()` filter. The reason we can't just change `add()` is it will be a breaking change. In addition, an `add` is still valuable between two inputs whereas sum is a bit different.

Reply:
@VedRatan yes, `add` should not be changed, create a new `sum` filter instead.